examples/profile-catalog-source.yaml

```diff
@@ -7,14 +7,24 @@ spec:
     - name: weaveworks-nginx
       description: This installs nginx.
       version: v0.1.0
+      tag: weaveworks-nginx/v0.1.0
       url: https://github.com/weaveworks/profiles-examples
       maintainer: weaveworks (https://github.com/weaveworks/profiles)
       prerequisites:
       - Kubernetes 1.18+
     - name: bitnami-nginx
       description: This installs some other nginx.
       version: v0.0.1
+      tag: bitnami-nginx/v0.0.1
       url: https://github.com/weaveworks/profiles-examples
       maintainer: weaveworks (https://github.com/weaveworks/profiles)
       prerequisites:
       - Kubernetes 1.19+
+    - name: nginx
+      description: This installs some other other nginx.
+      version: v1.0.0
+      tag: v1.0.0
```

Comment:
I think this could be removed, but for now its simplest to leave this. Tag/Version definitely seem a bit too repetitive